src/components/AppNavbar.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import { EnvelopeIcon } from '@heroicons/react/24/outline';
+import { BookOpenIcon, EnvelopeIcon } from '@heroicons/react/24/outline';
 import { TableCellsIcon } from '@heroicons/react/24/solid';
 import { User } from './User';
 
 const AppNavbar = () => (
   <nav className="App-navbar">
     <ul>
+      {process.env.REACT_APP_SHOW_STORYBOOK === 'true' && (
```

Comment:
Oof! `true` as a string gives me hives. (But, I know env vars are always strings.) No changes requested here, this is Correct, it's just also in a larger way Incorrect. 😆